TheFourFifths/consus-client
# Distribution

### Summary

Adding github releases and packaging the electron application

### Checklist


Finally, is it ready to be reviewed and merged:  🍏   

### How to Review

1. run ```npm run package```
2. make sure the macos and win64 projects are built and compressed into zipped folders (I couldn't build macos properly because of symlink issues with Windows.  If you're using Windows, this may not build and that's fine, but it should still create and zip and empty folder)
3. make sure the linux project is built and compressed into a .tgz
4. review the .travis.yml to make sure all configurations look reasonable for deployment on github

### Links

- https://msoese.atlassian.net/browse/TFF-47
